spec/classes/client_spec.rb

```diff
@@ -428,7 +428,7 @@
     context 'where media_server is set to a valid value' do
       let :params do
         {
-          :media_server => 'my_media_server',
+          :media_server => ['my_media_server'],
```

Comment:
Changining the type from string to array would break backward compatibility.

If you want to add functionality to use arrays to specifiy multiple hosts, please convert strings to an array with any2array() [1] to ensure proper handling in the template later on. Think [2] could give you some inspiration.

[1] https://github.com/puppetlabs/puppetlabs-stdlib#any2array
[2] https://github.com/ghoneycutt/puppet-module-rsyslog/blob/89836714b8e3b15b51997fede71e0e76ba278e45/manifests/init.pp#L447-L457